linode/manager
# refactor: [M3-8641] - Move Box component to ui package (Part 2)

## Description 📝
Part 2 of moving `Box` to the `ui` package. (Since this was branched off of m3-8641, it also includes those changes - the diff will get smaller once part 1 is merged. However, if it's easier to just check for everything on this PR, rather than splitting up the PRs, I can close part 1!)

will resolve merge conflicts + clean this up after part 1 is merged in

## Changes  🔄
- Move Box to ui package
- update rest of imports
- consolidate double imports to @linode/ui to just one import, if some files were already importing from the ui package

## How to test 🧪
Make sure there's no regressions in overall styling/layout in these specific areas:
- Notification Center
- Object storage
- Placement Groups
- Profile
- Stackscripts
- Support
- Top Menu
- Users
- Volumes
- VPCs
- double check the flows mentioned in part 1 throughout rest of CM 

## As an Author I have considered 🤔

*Check all that apply*
